obj/Debug/net6.0/backend.csproj.CoreCompileInputs.cache

```diff
@@ -1 +1 @@
-0cc89373bb98f415101a3152a5b7aa19b4facdc0
+0ed86dc1d2fb668c6b5b307dbe1d2c5f885c557f
```

Comment:
Tu acabou commitando os arquivos resultantes do processo de `build`
A gente usa o git pra que os arquivos fontes sejam guardados, mas não o projeto buildado
Pra ignorar certos arquivos a gente usa uma coisinha chamada `gitignore`
https://www.pluralsight.com/guides/how-to-use-gitignore-file
https://gist.github.com/vmandic/ac2ecc9c24f6899ee0ec46e4ce444a0e